qiwis.py

```diff
@@ -161,10 +166,9 @@ def __init__(
         super().__init__(parent=parent)
         self.appInfos = appInfos
         self.mainWindow = QMainWindow()
-        self.centralWidget = QWidget()
-        self.centralWidget.setStyleSheet("border-image: url(./resources/background.png);")
+        self.centralWidget = QMdiArea()
         self.mainWindow.setCentralWidget(self.centralWidget)
-        self._dockWidgets = defaultdict(list)
+        self._outerWidgets = defaultdict(list)

```

Comment:
What is the meaning of "outer widget"? I don't get the intuition from its name.
If there is no special reason, how about use the term "frame"?
For example, `self._frames`.
Also you can define a new type annotation for convenience: `Frame = Union[QMdiSubWindow, QDockWidget]`.
However, it is true that "frame" is not so precisely defined.

Reply:
I mean, it's a role to wrap up something.
Also, I agree with you that "frame" is not exact because there is a frame in `QMdiSubWindow` or `QDockWidget`, not in itself.

Thus, how about `self._wrapperWidgets`? I want to use "widget" because the common parent class of `QMdiSubWindow` and `QDockWidget` is `QWidget`.

Reply:
Okay then, please rename it to wrapper widgets!